javascriptfile/menu.js

```diff
@@ -2,6 +2,61 @@ const hamburger = document.querySelector('.hamburger_img');
 const closeHamburgerBtn = document.querySelector('.close_hamburger');
 const modalMenuContainer = document.querySelector('.modal_menu_container');
 const menuOption = document.querySelectorAll('.menu_option');
+const worksSection = document.querySelector('.works_section');
+
+const cardsData = [
+  {
+    id: 0,
+    name: ['Tonic', 'Tonic'],
+    description: [
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem Ipsum is simply dummy text of the printing and typesetting industry. A daily selection of privately personalized reads; no accounts or sign-ups required.', 'A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.'],
+    image: ['../img/jumbo1.jpg', '../img/jumbo5.png'],
+    technologies: ['html', ' css ', 'javaScript'],
+    technologiesDesk: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    technologiesPop: ['html', ' css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    Source: 'https://github.com/emekaemmanuel/res_portfolio',
+    liveLink: 'https://github.com/emekaemmanuel/res_portfolio',
+  },
```

Comment:
-  Your design looks really nice but kindly I suggest you add the size of your picture  as required by the previous reviewer, the picture must fit the content of the popup at 70% it will make your popup design on a desktop stick to the Figma design as required in the project :+1: 


## Your design 
![image](https://user-images.githubusercontent.com/95297251/209156228-4368eddc-dfcc-4e9a-9700-0599d0ea9410.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/209156117-d649221d-8260-4b53-a454-7cfa64635f29.png)